book/website/community-handbook/working-groups/working-groups-formalisation.md

```diff
@@ -0,0 +1,126 @@
+* **Discuss Previous Experience with WGs or Similar Organisational Formats**: What did you like about those formats, and what would you avoid? What kind of organisational histories do members of this WG come from?
+* **Discuss Initial Goals**: Are there concrete goals that your WG would like to accomplish at its founding? Have these goals been scoped appropriately based on resources, availability, and interest?
+    * Note: Frameworks like SMART goals, Agile, Kanban, Scrum methodology may be valuable here.
+* **Discuss the Group's Communication Plans**: Are there communication tools you might need for your ongoing work (Slack channel, GitHub organisation, emails)?
+
+### Logistics for Managing a Working Group
+
+Once a WG has been established, it is essential to identify and discuss the following topics within your group to establish its cadence and momentum:
+
+* **Establish Regular Meeting Times and/or Other Group Communication Needs**: How often should the group meet? Would they like to use an existing community call (such as Collaboration Cafe) for real-time discussions, or develop a separate call for co-working?
+* **Establish a Timeline for the WG**: How long should this group be operational? Should it be time-bound or ongoing indefinitely?
+* **Agree on Terms for All Roles**: How long should people be appointed in their roles, and how often should these roles change?
+* **Use GitHub Repository for Organising and Documenting WG Activities**: All WG activities (including notes and the charter, mentioned below) should be archived in a Turing Way GitHub repository (you can use and adapt [this template](https://github.com/the-turing-way/working-group-template)).
+    * Note: In "WG Resources," other infrastructure recommendations and needs (permissions, templates, and checklists) are available to help with this process.
+* **Establish a Reporting Mechanism and/or Public Notes Archive**: How will you share meeting notes and group progress with the wider community?
+    * Note: All WGs should upload notes to their respective GitHub repository [in *The Turing Way* organisation](https://github.com/the-turing-way/the-turing-way/). For a template, please see this template repository.
+* **Establish Appropriate Permissions for Accounts**: What accounts does this group need access to? Does this group have the proper account access and permissions to carry out its work?
+    * Note: The Community Manager and/or Project Manager can assist with this!
+* **Create a WG Charter**: Please follow the {ref}`Working Groups Charter template<ch-gov-wg-charter-template>`.
+    * Note: All WGs should openly share an updated charter to their respective GitHub repository [in *The Turing Way* organisation](https://github.com/the-turing-way/the-turing-way/).
+* **Create an Open Calendar Invite**: This may require support from a core delivery staff member to add the information to the public Google Calendar.
```

Comment:
```suggestion
* **Create an Open Calendar Invite**: This may require support from a member of the Community Management Working Group to add the information to the public Google Calendar.
```